app/Http/Controllers/ServiceController.php

```
@@ -137,27 +141,72 @@ public function disable(Server $server, Service $service): RedirectResponse
 
-        app(Uninstall::class)->uninstall($service);
+        return back()->with('info', app(RefreshServices::class)->refresh($server)
+            ? __('Refreshing services…')
+            : __('A services refresh is already running.'));
+    }
 
-        return back()->with('warning', __(':service is being uninstalled.', [
-            'service' => $service->name,
-        ]));
+    #[Get('/{service}/networking', name: 'services.networking')]
+    public function networking(Server $server, Service $service): JsonResponse
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$server'. (undefined)

(UnusedFormalParameter)